jf_agent/validation.py

```diff
@@ -58,23 +67,32 @@ def full_validate(config, creds):
     # Check for Git configs
     if config.git_configs:
         try:
-            git_success = validate_git(config, creds)
+            git_healthcheck_result = validate_git(config, creds, jellyfish_endpoint_info.git_instance_info)
+
         except Exception as e:
             print(f"Failed to validate Git due to exception of type {e.__class__.__name__}!")
 
             # Printing this to stdout rather than logger in case the exception has any sensitive info.
             print(e)
 
-
     else:
         logger.info("\nNo Git configs provided, skipping Git validation...")
 
     # Finally, display memory usage statistics.
     validate_memory(config)
 
+    healthcheck_result: IngestionHealthCheckResult = IngestionHealthCheckResult(ingestion_type=IngestionType.AGENT,
+                                                                                git_healthcheck_result=git_healthcheck_result,
+                                                                                jira_healthcheck_result=jira_healthcheck_result)
```

Comment:
Do we have black set up as a git hook? This is obviously a formatting NIT, I'm just surprised black left it like this